css/index.css

```diff
@@ -1,7 +1,15 @@
 /*
   Put css style here
  */
-h1 {
-  font-family: helvetica neue;
-  font-weight: 100;
+body {
+  font-family: Arial;
+}
+
+.glyphicon {
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.personalpoints {
```

Comment:
Great start on your CSS! `.personalpoints` is a little hard to read when written like that. Maybe try something like `.personal-points`?